.rhcicd/sonarqube/Dockerfile

```diff
@@ -9,6 +9,14 @@ ARG cacerts_keystore_password
 # The argument that holds Red Hat IT's custom certificate's location.
 ARG rh_it_root_ca_cert_url
 
+# install postgresql 11
```

Comment:
Install posgresql inside the test container to avoid starting a container inside a constainer